src/lib/components/multiDropzone/style.module.scss

```diff
@@ -0,0 +1,29 @@
+@use "../../scss/public/grid" as *;
+@use "../../scss/public/colors" as *;
+
+.container {
+  background-color: white;
+  padding: 24px;
+  box-shadow: 0px 2px 3px rgba(38, 38, 46, 0.04);
+  border-radius: 8px;
+
+  @include p-size-mobile {
+    padding: 16px;
+  }
+}
```

Comment:
I think the outline must be set somewhere so that it fits our guideline.

![image](https://user-images.githubusercontent.com/3629231/97594975-83006f80-1a03-11eb-9077-363f5ba93c7c.png)

Look at the outline set on a button for a good example

Reply:
btw: not sure the outline needs to be set on the `container`